core/application/src/state/executor/epoch_change.rs

```diff
@@ -751,14 +754,20 @@ impl<B: Backend> StateExecutor<B> {
         };
 
         // Calculate the epoch end timestamp.
+        let old_epoch_end_timstamp = self.committee_info.get(&epoch).unwrap().epoch_end_timestamp;
         let epoch_time = self.get_epoch_time();
-        let epoch_end_timestamp =
-            self.committee_info.get(&epoch).unwrap().epoch_end_timestamp + epoch_time;
+        let new_epoch_end_timestamp = old_epoch_end_timstamp + epoch_time;
+
+        // Calculate the time interval.
+        let time_interval = (new_epoch_end_timestamp - old_epoch_end_timstamp)
```

Comment:
Can't we just do replace `new_epoch_end_timestamp - old_epoch_end_timstamp` with `epoch_time`?
`new_epoch_end_timestamp - old_epoch_end_timstamp = old_epoch_end_timstamp + epoch_time - old_epoch_end_timstamp = epoch_time`